### PR TITLE
Bump Storm appVersion to 1.2.3

### PR DESCRIPTION
**Pull request by stackedsax**
While I was bumping the appVersion to Storm 1.2.3, I removed some cruft sitting in the siembol folder and upgraded the Storm Chart to use apiersion v2.  The only real difference for this chart is that requirements.yaml and requirements.lock is now inside Chart.yaml and Chart.lock

### CODE REVIEW

File: siembol/values.yaml [hunk 1]
```diff
@@ -1,28 +0,0 @@
-nginx-ingress:
```
Comment (jgiannuzzi) [inline siembol/values.yaml:1]: This has nothing to do with the upgrade of Storm 😅

File: siembol/dev.md [hunk 2]
```diff
@@ -1,6 +0,0 @@
-Some random notes for development
```
Comment (jgiannuzzi) [inline siembol/dev.md:1]: This has nothing to do with the upgrade of Storm 😅

File: siembol/values.yaml (hunk 1 above)
Comment (stackedsax) [reply siembol/values.yaml:1]: Sure, I can remove these.  Was just cleaning up some cruft...

Comment (jgiannuzzi) [reply siembol/values.yaml:1]: We should definitely clean those up. I'll open a separate PR for it.

Comment (jgiannuzzi) [reply siembol/values.yaml]: I created PR #10 to clean up the cruft, and I force pushed on this branch to clean up the clean up.

**Pull request merged**